src/app/view/components/login/dot-login-component/dot-login.component.ts

```diff
@@ -40,6 +40,7 @@ export class DotLoginComponent implements OnInit, OnDestroy {
     ) {}
 
     ngOnInit() {
+        this.dotMessageService.init('default');
```

Comment:
Why don't we set the default inside the service.

Reply:
is does by default, without params, but when we send the parameter we "force" to make a request and refresh the local storage